src/uwtools/tests/fixtures/hello_workflow_tags.yaml

```diff
@@ -15,3 +15,32 @@
       ACCOUNT: myaccount
       FOO: test.log
     log: /some/path/to/&FOO;
+    tasks:
+      task_hello:
+        attrs:
+          cycledef: howdy
+        account: "&ACCOUNT;"
+        command: "echo hello $person"
+        walltime: 00:01:00
+        memory: 512M
+        deadline: 202209290000
+        envar:
+          name: siri
+        join: hello
```

Comment:
Still working on making join/stderr/stdout optional and nodes/cores/native required